Resources/Prototypes/_CD/Entities/Clothing/OuterClothing/coats.yml

```
@@ -67,8 +67,8 @@
 - type: entity
   parent: ClothingOuterCoatEuropanVet
   id: ClothingOuterCoatEuropanSeparatist
-  name: separatist jacket
-  description: A heavy leather jacket, winterized and waterproofed for harsh underwater environments. Emblazoned with a Separatist patch on the shoulders.
+  name: independance jacket
```

```suggestion
  name: independence jacket
```

minor spelling mistake